streams/src/main/java/org/apache/kafka/streams/kstream/internals/AbstractStream.java

```diff
@@ -46,7 +46,7 @@
     protected final String name;
     protected final Serde<K> keySerde;
     protected final Serde<V> valSerde;
-    protected final Set<String> sourceNodes;
+    protected final Set<String> subTopologySourceNodes;
```

Comment:
Renaming this variable in this class and all its sub-classed to make it more clear what it is for